CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## Release v2.2.0 - 2020/09/01
+
+### Enhancement
+* `object`: Implement S3 api 'deleteBucketPolicy' and related test cases. [#757](https://github.com/chubaofs/chubaofs/pull/757)
+* `master`: Add interface to expand/shrink capacity of volume, capacity must be set more than 20% lager than used. [#764](https://github.com/chubaofs/chubaofs/pull/764)
+* `master`: Provide update **DataNode** **MetaNode** API, check invalid **DataNode**. [#813](https://github.com/chubaofs/chubaofs/pull/813)  
+* `cli`: Introduce log to cli tool, and added several commands to call master api. [#764](https://github.com/chubaofs/chubaofs/pull/764) [#801](https://github.com/chubaofs/chubaofs/pull/801)
+* `compile`: Support **arm64** cross compiler using docker; support **Ububtu 14.04+**, **CentOS7.6+**. [#779](https://github.com/chubaofs/chubaofs/pull/779)
+* `client`: Enable read from nearest **DataNode** IP. [#810](https://github.com/chubaofs/chubaofs/pull/810)  
+* `client`: Update volume follower read config from master periodicity. [#837](https://github.com/chubaofs/chubaofs/pull/837)  
+
+### Refactor
+* `object`: listObject and listObjectsV2 list result is not expected. [#769](https://github.com/chubaofs/chubaofs/pull/769)
+* `master`: Check whether the used space between the replicas is consistent when performing the load partition operation. [#813](https://github.com/chubaofs/chubaofs/pull/813)
+* `metanode` `datanode`: Optimize batch delete Extent; add **autoRepairLimitRater** on **DataNode**. [#781](https://github.com/chubaofs/chubaofs/pull/781)
+* `datanode`: Add **autoRepair** limit on **DataNode**. [#842](https://github.com/chubaofs/chubaofs/pull/842)  
+* `datanode`: **repairTask** skip the extent which has been deleted. [#842](https://github.com/chubaofs/chubaofs/pull/842) 
```

Comment:
It is recommended to modify to "`The data repair task will skip extent which has been already deleted`".